### DISCUSSION TITLE
Close a split

### DISCUSSION THREAD

**Discussion by dalvand** [kovidgoyal/kitty#7159] [General]
Is it possible to close a split with hotkey? Let's say I have a code running in it that is not responding and I need to kill it. None of the Ctrl-c or Ctrl-z strop the command and I need the close the entire split. Thanks.

**Comment by Luflosi** [comment]
Does https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Close-window work for you? Note that "window" in the documentation refers to kitty windows and not OS windows.

**Reply by dalvand** [reply]
Yes, that's exactly what I need. Thank you
